### ISSUE TITLE
[BUG] ctx.selected_labels is empty in resolve_input and resolve_output

### ISSUE THREAD

**Issue by msamadony** [voxel51/fiftyone#3557]
### System information

-   **OS Platform and Distribution**: Ubuntu 20.04
-   **Python version**: 3.8.10
-   **FiftyOne version**: 0.22.0
-   **FiftyOne installed from**: pip

### Commands to reproduce

See the sample operator in the code section below.

### Describe the problem

The `ctx.selected_labels` always return an empty list when it is read from the context of `resolve_input` and `resolve_output` of an operator in python. The `ctx.selected_labels` returns the correct information only when it is read from the `execute` function. It is worth noting that the value of `ctx.selected` works correctly in the three functions.

The below screenshots shows the input and output panels of the below referenced sample operator.

Operator Input:
![image](https://github.com/voxel51/fiftyone/assets/224718/17337a15-3832-4c60-9ca6-d48d9da62dec)

Operator Output:
![image](https://github.com/voxel51/fiftyone/assets/224718/768d7485-c4b3-46b7-90e3-fd98acc24c22)

Having a look at the frontend code, it seems that the `selectedLabels` state is missing from `useExecutionContext`:

https://github.com/voxel51/fiftyone/blob/7ee780bc7a4c854371dc9c5facd181517a718d9b/app/packages/operators/src/state.ts#L113-L132

### Code to reproduce issue

Below is a sample operator to reproduce the behavior:
```
class SelectedLabelsOperator(foo.Operator):
    @property
    def config(self):
        return foo.OperatorConfig(
            name="selected_labels_operator",
            label="Selected Labels Operator",
            dynamic=True,
        )

    def resolve_input(self, ctx):
        inputs = types.Object()

        inputs.message(
            "Number of Selected Labels",
            label=f"Number of Selected Label: {len(ctx.selected_labels)}",
        )

        inputs.message(
            "Number Selected Samples",
            label=f"Number Selected Samples: {len(ctx.selected)}",
        )

        return types.Property(inputs)

    def execute(self, ctx):
        return {
            "num_selected_labels": len(ctx.selected_labels),
            "num_selected_samples": len(ctx.selected),
        }

    def resolve_output(self, ctx):
        outputs = types.Object()

        outputs.message(
            "Number of Selected Labels",
            label=f"Number of Selected Label: {len(ctx.selected_labels)}",
        )

        outputs.message(
            "Number Selected Samples",
            label=f"Number Selected Samples: {len(ctx.selected)}",
        )

        outputs.int("num_selected_labels", label="Num Selected Labels")
        outputs.int("num_selected_samples", label="Num Selected Samples")

        return types.Property(outputs)

```

### Other info/logs

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Please do not use screenshots for sharing text. Code snippets should be used instead when providing tracebacks, logs, etc.

### What areas of FiftyOne does this bug affect?

-   [x] `App`: FiftyOne application issue
-   [ ] `Core`: Core Python library issue
-   [ ] `Server`: FiftyOne server issue

### Willingness to contribute

The FiftyOne Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the FiftyOne codebase?

-   [ ] Yes. I can contribute a fix for this bug independently
-   [ ] Yes. I would be willing to contribute a fix for this bug with guidance from the FiftyOne community
-   [ ] No. I cannot contribute a bug fix at this time

**Comment by ritch** [comment]
@msamadony Thank you for the very detailed bug report. I've submitted a PR to fix this #3558.

**Comment by msamadony** [comment]
@ritch Thanks a lot for the very quick feedback and fix. I left you one comment/question there, as I suspect there is another line that needs to be updated as well.
